Assets/Scripts/CustomerManager.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CustomerManager : MonoBehaviour
+{
+    [SerializeField] private GameObject customerPrefab;
+    [SerializeField] List<Transform> waypoints;
+
+    private List<GameObject> customers;
+    public List<GameObject> Customers 
+    {
+        get => customers;
+        set => customers = value;
+    }
+
+    public static CustomerManager Instance;
+
+    void Awake()
+    {
+        if (Instance != null)
```

Comment:
This check is good, another thing you could do is check if the `CusomerManager != this` just in case it gets assigned twice and then warns because of it.